internal/tenant/tenant.go

```diff
@@ -62,3 +63,38 @@ func (t tenantService) GetTenantInfoByNamespace(apiURL string, ns string) (InfoL
 
 	return tenantInfo, nil
 }
+
+// returns true if the cluster the ns is on has reached maximum capacity
+func (t tenantService) HasReachedMaxCapacity(apiURL, ns string) (bool, error) {
+
+	ti, err := t.GetTenantInfoByNamespace(apiURL, ns)
+	if err != nil {
+		return true, err
+	}
+
+	if len(ti.Errors) != 0 {
```

Comment:
> should all the errors not be concatenated instead of returning just the first one?

I think that is probably happening. err is the concatenated error from all errors in ti.Errors. @sthaha Can you make sure of this?